pkg/transcriptiondata/eurasian.tsv

```diff
@@ -1,20 +1,20 @@
 BIPA_GRAPHEME	CLTS_NAME	GENERATED	EXPLICIT	GRAPHEME	URL	LATEX	FEATURES	SOUND	IMAGE	COUNT	NOTE
-<NA>	<NA>		+	aɰ	http://eurasianphonology.info/search_exact?dialects=True&query=a%C9%B0					1	
-<NA>	<NA>		+	ä̠ː	http://eurasianphonology.info/search_exact?dialects=True&query=a%CC%88%CC%A0%CB%90					1	
+aɯ	<NA>		+	aɰ	http://eurasianphonology.info/search_exact?dialects=True&query=a%C9%B0					1	

```

Comment:
I suggest to add the "consonantal" little arc beyond the second element of the dipthong, as we do also in other cases, where we translate things like `aj`.